atlasdb-config/src/main/java/com/palantir/atlasdb/factory/PersistentStorageFactories.java

```diff
@@ -28,19 +29,24 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import com.google.common.base.MoreObjects;
+import com.google.common.collect.ImmutableList;
 import com.palantir.logsafe.Preconditions;
 import com.palantir.logsafe.SafeArg;
 
 public final class PersistentStorageFactories {
+    private static final Logger log = LoggerFactory.getLogger(PersistentStorageFactories.class);
     private static final Pattern UUID_PATTERN = Pattern.compile(
             "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$");
 
     private static final Set<String> SANITIZED_PATHS = new HashSet<>();
```

Comment:
static mutable state...???

Reply:
Yes this very sad, I'll have to refactor this in a separate PR.

Reply:
Curious how this ended up on develop :)